x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/endpoint_activity_log.tsx

```diff
@@ -32,9 +57,20 @@ export const EndpointActivityLog = memo(
           <>
             <SearchBar onSearch={onSearch} placeholder={i18.SEARCH_ACTIVITY_LOG} />
             <EuiSpacer size="l" />
-            {endpointActions.data.map((endpointAction) => (
-              <LogEntry key={endpointAction.action_id} endpointAction={endpointAction} />
-            ))}
+            {activityLoading ? (
+              <EuiLoadingContent lines={3} />
+            ) : (
+              // @ts-ignore
```

Comment:
what this ts-ignore hide? If this is needed, could you add next to it or below a comment with the hidden error?

Reply:
fixed the ts error in bfc8e964f4814c62a91113b7474b998b335b0796. It was complaining that some of the types were `unknown` or `any`